src/main/java/lotto/LottoDrawer.java

```diff
@@ -0,0 +1,43 @@
+package lotto;
+
+import java.util.List;
+
+public class LottoDrawer {
+    private final LottoMachine lottoMachine;
+    private final List<Integer> winningLottoNumbers;
+    private final int bonusNumber;
+
+
+    public LottoDrawer(LottoMachine lottoMachine, List<Integer> winningLottoNumbers, int bonusNumber) {
+        this.lottoMachine = lottoMachine;
+        this.winningLottoNumbers = winningLottoNumbers;
+        this.bonusNumber = bonusNumber;
+    }
+
+    private void checkWinningNumbers() {
+        for (List<Integer> lottos:lottoMachine.getLottoNumbers()){
+            int winningCount=0;
+            int bonusCount=0;
+
+            LottoResult lottoResult =
+                    new LottoResult(winningCountResult(lottos,winningCount),bonusCountResult(lottos,bonusCount));
+            lottoResult.checkWinningLotto();
+        }
+    }
+
+    private int winningCountResult(List<Integer> lottos, int winningCount) {
+        for (Integer lotto : lottos) {
+            winningCount += (int) winningLottoNumbers.stream().filter(e -> e.equals(lotto)).distinct().count();
+        }
+        return winningCount;
+    }
```

Comment:
주로 stream을 사용할 때는 `.` 단위로 엔터를 넣어 가독성을 높입니다.
```suggestion
    private int winningCountResult(List<Integer> lottos, int winningCount) {
        for (Integer lotto : lottos) {
            winningCount += (int) winningLottoNumbers.stream()
            .filter(e -> e.equals(lotto))
            .distinct()
            .count();
        }
        return winningCount;
    }
```

Reply:
반영해서 리팩토링 하겠습니다. 감사합니다!